test/e2e_node/docker_test.go

```diff
@@ -70,4 +78,170 @@ var _ = framework.KubeDescribe("Docker features [Feature:Docker]", func() {
 			}
 		})
 	})
+
+	Context("when live-restore is enabled [Serial] [Slow] [Disruptive]", func() {
+		It("containers should not be disrupted when the daemon is down", func() {
+			const (
+				podName           = "live-restore-test-pod"
+				containerName     = "live-restore-test-container"
+				volumeName        = "live-restore-test-volume"
+				timestampFilename = "timestamp"
+			)
+
+			isSupported, err := isDockerLiveRestoreSupported()
+			framework.ExpectNoError(err)
+			if !isSupported {
+				framework.Skipf("Docker live-restore is not supported.")
```

Comment:
Offline discussed. This is my misunderstanding. `Skip` will panic and recover, thus could skip the test.